src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -42,6 +42,33 @@
  */
 class CommonController extends FrameworkBundleAdminController
 {
+    /**
+     * Get a summary of recent events on the shop.
+     * This includes:
+     * - Created orders
+     * - Registered customers
+     * - New messages.
+     *
+     * @return JsonResponse
+     */
+    public function notificationsAction()
+    {
+        // TODO: Use CQRS
```

Comment:
in a new PR or TODO for this one?

Reply:
In another PR, or in this one if somebody is available to do it now. :)

Reply:
See #12870